src/Components/WebCollapse.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react"
+
+export class WebCollapse extends React.Component<{ open: boolean }> {
+  wrapperRef: HTMLDivElement | null = null
+  onTransitionEnd = () => {
+    if (!this.wrapperRef) {
+      return
+    }
+    this.wrapperRef.style.height = this.props.open ? "auto" : "0px"
+  }
+
+  componentDidMount() {
+    if (!this.wrapperRef) {
+      return
+    }
+
+    this.wrapperRef.addEventListener("transitionend", this.onTransitionEnd)
+    this.wrapperRef.style.height = this.props.open ? "auto" : "0px"
+  }
+
+  componentDidUpdate() {
+    if (!this.wrapperRef) {
+      return
+    }
+    if (this.props.open && this.wrapperRef.style.height === "0px") {
+      // open
+      this.wrapperRef.style.height = "auto"
+      const actualHeight = this.wrapperRef.offsetHeight
+      this.wrapperRef.style.height = "0px"
+      setTimeout(() => {
+        this.wrapperRef.style.height = actualHeight + "px"
```

Comment:
Will this give us an issue if we add height to the expanded component (i.e. an error message under the input?). If so we may need to re-think.

I'm also wondering if the name of this component can be a bit more descriptive-- maybe `StaticCollapse`? Or is the reason it's called `WebCollapse` because it uses styles/offsets that are not available in RN? Since reaction is only used on the web it sounds a little confusing in this context.